src/main/java/io/confluent/connect/jdbc/source/JdbcSourceConnectorConfig.java

```diff
@@ -1439,6 +1495,33 @@ public List<String> incrementingColMappingRegexes() {
         .collect(java.util.stream.Collectors.toList());
   }
 
+  /**
+ * Get the query string from either query or query.masked config.
+ * Prioritizes query.masked over query if both are set (though validation should prevent this).
```

Comment:
If there is a validation in place, there should never be a situation where both are set. Update the comment.